evergreen.yml

```diff
@@ -65,17 +65,23 @@ functions:
         shell: bash
         working_dir: mongo-powerbi-connector
         script: |
-          export MONGO_POWERBI_CONNECTOR_VERSION=$(git describe --always | sed 's/^v//g')
           export UPDATE_PLUGIN_VERSION=true
           git describe --exact-match HEAD || UPDATE_PLUGIN_VERSION=false
```

Comment:
Let's move `UPDATE_PLUGIN_VERSION=false` into the below `else` in line 74 so that it is clearer that we will not be updating the release version in `MongoDBAtlasODBC.pq` to the version `test_release`.  No need to base it on `git describe --exact-match HEAD` since we're already checking for whether it's a tag release there.